hedyweb.py

```diff
@@ -28,7 +28,10 @@ def get_translations(self, language):
 class PageTranslations:
   def __init__(self, page):
     self.data = {}
-    translations = glob.glob('content/pages/' + page + '/*.yaml')
```

Comment:
It seems we don't cache this (yet). Might be one reason for the extra memory load we currently have, doesn't have to be fixed in this PR but it would be nice if we cache the page translations as well.